src/main/java/org/openstreetmap/atlas/generator/AtlasGeneratorJobGroup.java

```diff
@@ -0,0 +1,110 @@
+package org.openstreetmap.atlas.generator;
+
+import org.apache.hadoop.mapred.lib.MultipleOutputFormat;
+import org.openstreetmap.atlas.generator.persistence.MultipleAtlasCountryStatisticsOutputFormat;
+import org.openstreetmap.atlas.generator.persistence.MultipleAtlasOutputFormat;
+import org.openstreetmap.atlas.generator.persistence.MultipleAtlasProtoOutputFormat;
+import org.openstreetmap.atlas.generator.persistence.MultipleAtlasStatisticsOutputFormat;
+import org.openstreetmap.atlas.generator.persistence.delta.RemovedMultipleAtlasDeltaOutputFormat;
+import org.openstreetmap.atlas.geography.atlas.Atlas;
+import org.openstreetmap.atlas.geography.atlas.delta.AtlasDelta;
+import org.openstreetmap.atlas.geography.atlas.statistics.AtlasStatistics;
+
+/**
+ * @author samgass
+ */
+public enum AtlasGeneratorJobGroup
+{
+    RAW(0, "Raw Atlas Creation", "rawAtlas", Atlas.class, MultipleAtlasOutputFormat.class),
+    LINE_SLICED(
+            1,
+            "Line Sliced Atlas Creation",
+            "lineSlicedAtlas",
+            Atlas.class,
+            MultipleAtlasOutputFormat.class),
+    LINE_SLICED_SUB(
+            2,
+            "Line Sliced Sub Atlas Creation",
+            "lineSlicedSubAtlas",
+            Atlas.class,
+            MultipleAtlasOutputFormat.class),
+    FULLY_SLICED(
+            3,
+            "Fully Sliced Atlas Creation",
+            "fullySlicedAtlas",
+            Atlas.class,
+            MultipleAtlasOutputFormat.class),
+    WAY_SECTIONED(
+            4,
+            "Way Sectioned Atlas Creation",
+            "atlas",
+            Atlas.class,
+            MultipleAtlasOutputFormat.class),
+    WAY_SECTIONED_PBF(
```

Comment:
Is this one a duplicate?

Reply:
different output format! MultipleAtlasOutputFormat vs MultipleAtlasProtoOutputFormat

Reply:
ooooh!